scripts/common.ts

```diff
@@ -57,3 +57,7 @@ export function getTKey(arg: Node): string | null {
     }
     return null;
 }
+
+export function isTranslation(translation: Translations[string]): translation is PluralisedTranslation {
+    return typeof translation === "object" && "one" in translation && "other" in translation;
```

Comment:
is `one` strictly required for a valid PluralisedTranslation? The type definition implies not.